src/controllers/IBackendController.ts

```diff
@@ -0,0 +1,20 @@
+import { Express } from "express";
+import { ParamsDictionary, RequestHandler } from "express-serve-static-core";
+
+/*
+ * Standard interface for Backend controllers
+ */
+export interface IBackendController {
```

Comment:
Also here, we know we are in the context of a backend app - perhaps since this is an Express controller you can call them `IExpressController`

Reply:
I agree, this name is really bad. 
But since "Express" is the name of the framework we use, wouldn't it be better to rename it to "IBaseController" (or something similar) to avoid confusion? This interface in future can group other common functions we want our controllers to implement.